codespell_lib/data/dictionary.txt

```diff
@@ -12219,6 +12222,8 @@ explitit->explicit
 explitly->explicitly
 explizit->explicit
 explizitly->explicitly
+exploition->exploitation, exploit,
+exploitions->exploitation, exploit,
```

Comment:
My suggestions:
```
exploition->explosion, exploitation, exploit,
exploitions->explosions, exploitations, exploits,
```

Reply:
Did a follow-up PR for this suggestions in #1754